src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -128,6 +128,29 @@ static std::unordered_map<std::string, ov::PartialShape> get_variables_shapes(co
     return variables_shape_map;
 }
 
+template<typename T>
+static int64_t find_sink_position(const ov::SinkVector& sinks, const std::shared_ptr<T>& sink)
+    int64_t pos = 0;
+    for (const auto& s : sinks) {
+        if (s == sink) {
+            return pos;
+        }
+        pos++;
+    };
+    return -1;
+}
+
+static int64_t find_sink_position(const ov::SinkVector& sinks, const std::shared_ptr<const ov::op::Sink>& sink) {
```

Comment:
Please fix the `find_sink_position`.
Remove this one and check that everything works.

Reply:
I've removed the second function and checked that it works.